src/main.cpp

```diff
@@ -80,6 +80,9 @@ int main(int argc, char* argv[]) {
   auto rect = runner.getInteractionService()->getBasicInteractionRect(NovelRT::GeoVector<float>(200, 200), rectArgs);
   rect->subscribeToInteracted([]{novelChanRect->setActive(!novelChanRect->getActive());});
 
+  auto coreclrService = runner.getCoreClrService();
+  auto testFunction = coreclrService->getFunction<void()>(STR("ClassLibrary1.dll"), STR("ClassLibrary1.Class1, ClassLibrary1"), STR("Test"), STR("ClassLibrary1.TestSignature, ClassLibrary1"));
```

Comment:
ClassLibrary1.dll isn't included here, since it was just a test.

Before the PR is merged, we should determine how cmake will pull down any managed NuGet packages that are dependencies and replace this with using that instead...

Reply:
This was changed to a new `NovelRT.Dotnet` csproj which is checked in, but which requires a bit more infrastructure work to properly integrate with the build system.

Reply:
Just to flag this up, we should probably decide together what's handled by the SDK vs what's handled by the engine core in this aspect.

Specifically, should the SDK be handling `dotnet restore` instead as a whole, for example? Or did you want to split it out so that it only handles `dotnet restore` for end-user csproj files?